docs/software/installation.md

```diff
@@ -271,5 +271,118 @@ Once you managed to install Jam, make sure to understand how to use it.
 
 [:octicons-arrow-right-24: First Use][cheatsheet]
 
+### ... installing `jmwalletd`, `ob-watcher` and `jam` services (optional but recommended)
+The following will setup three services that run at startup, so you don't need to manually start Jam or any related daemon. That is, every time reboot your server, you can access Jam via the URL you have configured (e.g. 127.0.0.1:3000 or similar) without any further action.
+
+The service scripts below assume standard joinmarket structure, such as:  
+Working directory: `/home/joinmarket/.joinmarket`  
+Main joinmarket directory: `/home/joinmarket/joinmarket-clientserver`  
+User and group: `joinmarket` and `joinmarket`  
+
+#### Create these files with the content below each
+For the `jmwalletd.service`, create this file:
+```sh
+sudo nano /etc/systemd/system/jmwalletd.service
+```
+...and add this content
+```sh
+[Unit]
+Description=JoinMarket Wallet Daemon For Jam
+After=network.target
```

Comment:
What do you think about something like the following?
```yml
Requires=bitcoin.service
Requires=tor.service
After=bitcoin.service
After=tor.service
```

Reply:
Sounds reasonable. I'm not an expert on service scripts - I put the very basic stuff to get the service working.

The only thing I would have against your proposal is related a bit with UX. That is, if either Tor or Bitcoin are not working, then Jam wont start and the user sees nothing in the browser and needs to figure out what happened, maybe opening the log via console to the services and finding out why jam.service didn't start.

On the other hand, if we don't require Tor nor Bitcoin, the Jam service will start. If either Tor or Bitcoin aren't working, Jam will still open but 'scream in the face' of the user that some needed services are missing. This would save the user some forensics of why Jam is not working.

Reply:
I get what you mean, yeah. In this case, only `After` can be added.
However, I am fine either way. It is a nice improvement! :muscle: 

(One thing is, that if `tor` is not installed/working, Jam cannot display or indicate this to the user but most functionality is defective, e.g. Collaborative transactions won't work.)